UnsupportedOperationException throwing in Language Server for Liberty

LCLS output tab include the following exception:
```
Oct 15, 2024 2:44:21 PM org.eclipse.lsp4j.jsonrpc.RemoteEndpoint handleNotification
WARNING: Notification threw an exception: {
  "jsonrpc": "2.0",
  "method": "$/setTrace",
  "params": {
    "value": "off"
  }
}
java.lang.RuntimeException: java.lang.reflect.InvocationTargetException
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:67)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.notify(GenericEndpoint.java:152)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.handleNotification(RemoteEndpoint.java:220)
	at org.eclipse.lsp4j.jsonrpc.RemoteEndpoint.consume(RemoteEndpoint.java:187)
	at io.openliberty.tools.langserver.common.ParentProcessWatcher.lambda$apply$0(ParentProcessWatcher.java:154)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.handleMessage(StreamMessageProducer.java:194)
	at org.eclipse.lsp4j.jsonrpc.json.StreamMessageProducer.listen(StreamMessageProducer.java:94)
	at org.eclipse.lsp4j.jsonrpc.json.ConcurrentMessageProcessor.run(ConcurrentMessageProcessor.java:113)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:572)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:317)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1144)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:642)
	at java.base/java.lang.Thread.run(Thread.java:1595)
Caused by: java.lang.reflect.InvocationTargetException
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:118)
	at java.base/java.lang.reflect.Method.invoke(Method.java:586)
	at org.eclipse.lsp4j.jsonrpc.services.GenericEndpoint.lambda$null$0(GenericEndpoint.java:65)
	... 12 more
Caused by: java.lang.UnsupportedOperationException
	at org.eclipse.lsp4j.services.LanguageServer.setTrace(LanguageServer.java:125)
	at java.base/jdk.internal.reflect.DirectMethodHandleAccessor.invoke(DirectMethodHandleAccessor.java:103)
	... 14 more
```

--- a/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
+++ b/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
@@ -1,5 +1,5 @@
 /*******************************************************************************
-* Copyright (c) 2020, 2023 IBM Corporation and others.
+* Copyright (c) 2020, 2024 IBM Corporation and others.
 *
 * This program and the accompanying materials are made available under the
 * terms of the Eclipse Public License v. 2.0 which is available at

--- a/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
+++ b/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
@@ -127,8 +127,7 @@ public class LibertyLanguageServer extends AbstractLanguageServer implements Pro
 
     @Override
     public void setTrace(SetTraceParams params) {
-        LOGGER.warning("setTrace Reached");
-        // to avoid having UnsupportedOperationException, the method is implemented
+        // to avoid exception in vscode, the method is implemented
         // FIXME : implement the behavior of this method.
     }
 }

--- a/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
+++ b/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
@@ -21,6 +21,7 @@ import org.eclipse.lsp4j.InitializeParams;
 import org.eclipse.lsp4j.InitializeResult;
 import org.eclipse.lsp4j.InitializedParams;
 import org.eclipse.lsp4j.ServerCapabilities;
+import org.eclipse.lsp4j.SetTraceParams;
 import org.eclipse.lsp4j.TextDocumentSyncKind;
 import org.eclipse.lsp4j.services.LanguageClient;
 import org.eclipse.lsp4j.services.LanguageClientAware;
@@ -122,5 +123,12 @@ public class LibertyLanguageServer extends AbstractLanguageServer implements Pro
 
     public void setLanguageClient(LanguageClient languageClient) {
         this.languageClient = languageClient;
+    }
+
+    @Override
+    public void setTrace(SetTraceParams params) {
+        LOGGER.warning("setTrace Reached");
+        // to avoid having UnsupportedOperationException, the method is implemented
+        // FIXME : implement the behavior of this method.
     }
 }

--- a/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
+++ b/liberty-ls/src/main/java/io/openliberty/tools/langserver/LibertyLanguageServer.java
@@ -127,7 +127,7 @@ public class LibertyLanguageServer extends AbstractLanguageServer implements Pro
 
     @Override
     public void setTrace(SetTraceParams params) {
-        // to avoid exception in vscode, the method is implemented
+        // to avoid having UnsupportedOperationException, the method is implemented
         // FIXME : implement the behavior of this method.
     }
 }

